grid_documentation_changes (#306)
SeleniumHQ/seleniumhq.github.io: docs_source_files/content/getting_started/quick.en.md

of learning Selenium script syntax.

## Grid

> Selenium Grid is a feature in Selenium that allows you to run test cases in different machines across different platforms. The control of triggering the test cases is on the local machine, and when the test cases are triggered, they are automatically executed by the remote machine

Reviewer: Hi @rahuljhakant ,

Thanks for the PR. Can you please make sure to have a line width of 70-80

Thanks,
Harsha

> Selenium Grid is a feature in Selenium that allows you to run test cases in different machines across different platforms. The control of triggering the test cases is on the local machine, and when the test cases are triggered, they are automatically executed by the remote machine

Reviewer: ```suggestion
Selenium Grid allows you to run test cases in different machines across different platforms. 
The control of triggering the test cases is on the local end, and when the test cases are triggered, they are automatically executed by the remote end.
```

> The control of triggering the test cases is on the local end, and when the test cases are triggered, they are automatically executed by the remote end.

Reviewer: Hi @rahuljhakant ,

Thanks for the PR. There are couple of changes required to get this pr merged.

1. Limit each line with max of 70-80 characters in line width. For example
this line 'The control of triggering the test cases is on the local end, and when the test cases are triggered, they are' contains 109 chars, you can spilt into two lines

2. The changes are done for only english version document. Still there are 7 translated pages (french, de, zh-ch, ko, nl etc) the same text changes should be done in those .md pages too.

Please let us know if you still need help.

Regards,
Harsha

Author: @Harsha509  Now I got your point. I will do the translations for all the languages and also Limit each line with a max of 70-80 characters in line width.
